pkg/pgmodel/cache/inverted_labels_cache.go

```diff
@@ -0,0 +1,63 @@
+package cache
+
+import (
+	"fmt"
+
+	"github.com/dgraph-io/ristretto"
+)
+
+type LabelInfo struct {
```

Comment:
```suggestion
type LabelPos struct {
```

Reply:
Imho `labelInfo` is more appropriate as it contains both label id + position. Also it is in line with previous naming.